src/main/java/com/github/hey_world_team/currency_converter/controller/rest/CentralBankApiController.java

```diff
@@ -1,7 +1,42 @@
 package com.github.hey_world_team.currency_converter.controller.rest;
 
+import com.github.hey_world_team.currency_converter.config.PropertiesForFileService;
+import com.github.hey_world_team.currency_converter.service.FileService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.IOException;
+
 
 @RestController
 public class CentralBankApiController {
-}
+    private final FileService fileService;
+
+    @Autowired
+    public CentralBankApiController(FileService fileService) {
+        this.fileService = fileService;
+    }
+
+    //TODO for future api with current date
+    //private static final String DATE_API = "date_req";
+    private static final String LINK = "http://www.cbr.ru/scripts/XML_daily.asp";
```

Comment:
the link must be placed in properties and received via @Valuethe link must be placed in properties and received via @Value

Reply:
Done!